File: src/main/java/com/now/nowbot/service/messageServiceImpl/MultiplePlayService.java
package com.now.nowbot.service.messageServiceImpl;

import com.now.nowbot.qq.event.MessageEvent;
import com.now.nowbot.service.MessageService;
import org.jetbrains.annotations.NotNull;

//@Service("MP")
public class MultiplePlayService implements MessageService<String> {
    //TODO 这个干啥的？

    @Override
    public boolean isHandle(@NotNull MessageEvent event, @NotNull String messageText, @NotNull DataValue<String> data) throws Throwable {
        return false;
    }

    @Override
    public void HandleMessage(MessageEvent event, String data) throws Throwable {

    }
}

File: src/main/java/com/now/nowbot/service/messageServiceImpl/MultiplePlayService.kt
package com.now.nowbot.service.messageServiceImpl

import com.now.nowbot.qq.event.MessageEvent
import com.now.nowbot.service.MessageService
import com.now.nowbot.service.MessageService.DataValue

//@Service("MP")
class MultiplePlayService : MessageService<String> {
    //TODO 这个干啥的？
    override fun isHandle(
        event: MessageEvent, messageText: String, data: DataValue<String>
    ): Boolean {
        return false
    }

    override fun HandleMessage(event: MessageEvent, data: String) {
    }
}
